packages/dds/tree/src/feature-libraries/optional-field/index.ts

```diff
@@ -8,6 +8,7 @@ export { OptionalChangeset, RegisterId } from "./optionalFieldChangeTypes";
 export {
 	optionalChangeHandler,
 	optionalFieldEditor,
+	OptionalFieldEditor,
```

Comment:
This was done so I could add explicit types to some things, following what sequence field already did in terms of exports.